### PR TITLE
Remove Course.main.

### PR DESCRIPTION
**Pull request by spark008**
Yesss master :ghost:

### CODE REVIEW

File: app/mailers/session_mailer.rb [hunk 1]
```diff
@@ -30,8 +30,8 @@ def team_invite_email(athena, origin_id, team_id)
     @team_name = Team.find_by_id(team_id).name
     @partition_name = TeamPartition.find_by_id(Team.find_by_id(team_id).partition_id).name
     email = athena + "@mit.edu"
-    subject = "#{Course.main.number} Team Invite"
-    mail from: "#{Course.main.number} staff <#{Course.main.email}>", to: email,
+    subject = "Igor Team Invite"
```
Comment (pwnall) [inline app/mailers/session_mailer.rb]: Teams are attached to courses, so the subject here should include the course number and/or name.

**Comment by spark008** [comment]
For the team invitation email, I changed as little code as I could get away with since the teams feature code is untested and clearly needs some work.

File: test/fixtures/email_resolvers.yml [hunk 2]
```diff
@@ -28,14 +28,14 @@ public_ldap:
   use_ldaps: false
 
 # NOTE: MIT's LDAP service hangs indefinitely when not used from campus IPs.
```
Comment (pwnall) [inline test/fixtures/email_resolvers.yml]: If this isn't used in any test, just kill it.

**Comment by pwnall** [comment]
Thank you!

**Pull request merged**